types/actions/omfwd.pp

```diff
@@ -0,0 +1,27 @@
+type Rsyslog::Actions::Omfwd = Struct[{
+  Target                            => Optional[Variant[String[1], Stdlib::IP::Address]],
```

Comment:
An IP address is always going to be a non zero length string?

Reply:
Hmm, then I'll probably just stick to `String[1]`. The docs say it has to be in either `hostname:port`, `fqdn:port`, or `ip_address:port` format, BUT that port can be omitted :/

Reply:
Rsyslog is not the place to create a new data type alias for the `(hostname|fqdn|ip_address):port` format. That should probably go in Stdlib or Extlib